.rubocop.yml

```diff
@@ -0,0 +1,48 @@
+inherit_gem:
```

Comment:
this cant be here I think as it will do nothing. this needs to be in project/.rubocop.yml.

Reply:
Probably...I thought to test this out once we release the gem, cause neither of `rubocop-gems` that I've tried locally (by providing `path` in the Gemfile) worked.

[rubocop-rspec](https://github.com/rubocop-hq/rubocop-rspec/blob/master/.rubocop.yml) has a weird config where it requires its own gem name